src/mixins/filing-mixin.ts

```diff
@@ -28,12 +28,12 @@ export default class FilingMixin extends DateMixin {
   @State filingData!: Array<FilingDataIF>
   @State entityName!: string
 
+  @Getter getBusinessAddress!: OfficeAddressIF
   @Getter getCurrentDate!: string
-  @Getter getLegalType!: CorpTypeCd
-  @Getter getIdentifier!: string
   @Getter getFoundingDate!: Date
+  @Getter getIdentifier!: string
+  @Getter getLegalType!: CorpTypeCd
   @Getter getRegisteredOfficeAddress!: OfficeAddressIF
-  @Getter getBusinessAddress!: OfficeAddressIF
 
```

Comment:
Sort imports. Not needed just happened to come across this when trying to change something